src/dualscreen.tsx

```diff
@@ -355,6 +352,17 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
     );
   };
 
+  private _onActiveSlideChanged = (slideItem: ImageItem) => {
```

Comment:
Why do we use both "image" and "slide". Let's choose one terminology as I think this is confusing